sentences/en/lock_HassGetState.yaml

```diff
@@ -3,34 +3,42 @@ intents:
   HassGetState:
     data:
       - sentences:
-          - "is <name> {lock_states:state} [in <area>]"
+          - "(is|are) <name> ([<currently>];{lock_states:state} [in <area>])"
         response: one_yesno
         requires_context:
           domain: lock
         slots:
           domain: lock
 
       - sentences:
-          - "(is|are) [there] any door[s] {lock_states:state} [in <area>]"
-          - "(is|are) [there] any {lock_states:state} door[s] [in <area>]"
+          - "((is|are) [there]|do I have) any <lockable> ([<currently>];{lock_states:state} [in <area>])"
```

Comment:
Same thing, you can't have a completely optional permutation item. Please refactor and keep in mind the whitespace in optional parts.